codepropertygraph/src/main/scala/io/shiftleft/passes/ParallelIteratorExecutor.scala

```diff
@@ -1,11 +1,13 @@
 package io.shiftleft.passes
 
+import scala.collection.mutable
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration.Duration
 import scala.concurrent.{Await, Future}
+import scala.reflect.ClassTag
 
-class ParallelIteratorExecutor[T](iterator: Iterator[T]) {
-  def map[D](func: T => D): Iterator[D] = {
+class ParallelIteratorExecutor[T: ClassTag](iterator: Iterator[T]) {
```

Comment:
`ClassTag` doesn't seem to be needed here?

Reply:
Correct, I've removed it